src/database/DataTypes/NameTag.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import Realm from 'realm';
+
+export class NameTag extends Realm.Object {}
+
+NameTag.schema = {
+  name: 'NameTag',
+  primaryKey: 'id',
+  properties: {
+    id: 'string',
+    description: { type: 'string', default: 'Placeholder description' },
```

Comment:
Think the placeholder is better than optional - I don't really know when you're gonna see it, but good bug indicator!!